post.hbs

```diff
@@ -28,7 +28,20 @@ into the {body} of the default.hbs template --}}
             </header>
 
             {{#if feature_image}}
-            <figure class="post-full-image" style="background-image: url({{feature_image}})">
+            <figure class="post-full-image">
+                {{!-- This is a responsive image, it loads different sizes depending on device
+                https://medium.freecodecamp.org/a-guide-to-responsive-images-with-ready-to-use-templates-c400bd65c433 --}}
+                <img
+                    srcset="{{img_url feature_image size="s"}} 300w,
+                            {{img_url feature_image size="m"}} 600w,
+                            {{img_url feature_image size="l"}} 1000w,
+                            {{img_url feature_image size="xl"}} 2000w"
+                    sizes="(max-width: 800px) 400px,
+                            (max-width: 1170px) 700px,
+                            1400px"
```

Comment:
I know this already got approved and I'm not a team member, but as someone with some responsive image experience, I wanted to ask a few questions. Feel free to let me know if I've missed something obvious.

1) Based on my reading of your `sizes` attribute here, the `300w` asset will never be loaded. You've told the browser the image will be `400px` on small screens, so it'll load the next largest asset, `600w`.

2) I understand you've defined a standard set of image sizes, so they won't always match the exact rendering, but it looks like you used this 400, 700, 1400 scale in several places. Would it make sense to provide assets at 400, 700, and 1400px?

- until 800px, `sizes` says image will be `400px`, browser will load `600w` asset
- until 1170px, `sizes` says image will be `700px`, browser will load `1000w` asset
- above 1171px, `sizes` says image will be `1400px`, browser will load `2000w` asset

Also note that on retina displays, the browser will try to load 2x assets:

- until 800px, `sizes` says image will be `400px`, (800 retina) browser will load `1000w` asset
- until 1170px, `sizes` says image will be `700px`, (1400 retina) browser will load `2000w` asset
- above 1171px, `sizes` says image will be `1400px`, (2400 retina) browser will load `2000w` asset (which is below target width, but the largest asset provided)

Reply:
Quite honestly it's my first time implementing responsive images in this format and I've found the whole thing extremely confusing - so if you have an improvement to this PR (and its sounds like you do!) then any help would be extremely welcome 🤗

Reply:
kk - I'll see what I can do tomorrow :)

Reply:
Okay, one question before I try to make any code suggestions — is it a problem to add additional sizes to `image_sizes` in package.json? It won't hurt the user, because the browser will just download the appropriate resource, but I'm not sure what your image hosting provider is, or whether generating a few additional sizes for each image would kill your bandwidth costs. (This was a problem at my last job, and it's totally reasonable to strike a compromise between providing the perfect assets vs balancing costs.)

If more assets are fine, then I'd suggest adding 400, 700, 1400 assets to match what you're trying to load on the client.

If you'd prefer to avoid generating additional resources, then I'd look at redefining `s`, `m`, and `l` to 400, 700, and 1400, to better match the most common use cases.

Either way, I'm happy to submit a PR.